packages/docs/docs/rate-limits/index.md

```diff
@@ -14,26 +14,11 @@ All rate limits are calculated per IP address over a one minute window.
 
 Rate limits can be increased for paid plans. Please [contact us](mailto:info+rate-limits@medplum.com?subject=Increase%20rate%20limits) for more information.
 
-## FHIR Interaction Quota
+## Total Load Rate Limit
```

Comment:
I think there was a bit of back and forth on this what to call this. Since we are referring to these as FHIR Interactions  (in the headers and in code) we should probably keep that in the language here.